### ISSUE TITLE
Ouyru in 4-2 starting in "flying" mode.

### ISSUE THREAD

**Issue by Tofutoshi** [EdenServer/community#5924]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [x] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [x] I have provided my client version in the details. (type /ver into your game window)
- [ ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

verion 30181205_0

Ouryu in the CoP mission The Savage starts the fight evading all melee attacks until a mistmelt is used or enough time goes by.

![image](https://github.com/user-attachments/assets/f1fd85de-e1a1-4f09-93da-07022dddc6d3)

**Comment by Knack214** [comment]
Just ran this fight a few times and Ouryu goes into flying mode every time we used a WS. Which forced us to use a mistmelt every time we needed/wanted to WS. We tried to time our WS but unless Ouryu is stuck in animation it immediately went into the air before the 2nd WS registered. 
![ouryu WS](https://github.com/user-attachments/assets/bd14042e-2772-4c95-8afb-503d45ab6bf5)

**Comment by 59blargedy** [comment]
damage makes it change just like big ouryu, but at a much smaller threshold, so it's likely the WS is pushing it up, but i cant tell from the screenshot. how much damage was done before it popped up?

**Comment by Knack214** [comment]
That checks. We were averaging 750-800 with full brd/cor buffs. Guess we just got caught by surprise when we had to use 5-6 mistmelts per fight to kill it. The previous times we were fighting we used 2 at most spamming penta.

**Comment by 59blargedy** [comment]
yeah it was changed based on a capture last maintenance - it didn't have the damage component prior, was just time based. definitely more of a pita for melee strats now. 
the original issue is still an issue, but is resolved in next update

**Comment by 59blargedy** [comment]
resolved in October 9, 2024 update.
